test/suites/dims_tests.jl

```diff
@@ -0,0 +1,32 @@
+
+@testitem "dims" begin
+    u = lazy(fsprand(3, 1, 2, 1, 1, 0.2))
+    v = compute(Finch.dropdims(u, [2, 4, 5]))
+    u = compute(u)
+
+    @test size(v) == (3, 2)
+
+    not_equal = Scalar(0.0)
+    @finch begin
+        for i in _, j in _
+            not_equal[] += (v[i, j] != u[i, 1, j, 1, 1])
+        end
+    end
+
+    @test not_equal[] == 0.0
+
+    # u = lazy(fsprand(3, 1, 2, 1, 1, 0.2))
```

Comment:
@willow-ahrens I think this commented out case is a good example of something that takes a while. 

FYI: a long time == around < 5 minutes (which for me is longer than expected, especially for a test)